src/library/banshee/bansheeplaylistmodel.cpp

```diff
@@ -75,71 +75,75 @@ void BansheePlaylistModel::setTableModel(int playlistId) {
         m_playlistId = playlistId;
 
         QSqlQuery query(m_database);
-        QString strQuery("CREATE TEMP TABLE IF NOT EXISTS %1"
-            " (" CLM_TRACK_ID " INTEGER, "
-                 CLM_VIEW_ORDER " INTEGER, "
-                 CLM_ARTIST " TEXT, "
-                 CLM_TITLE " TEXT, "
-                 CLM_DURATION " INTEGER, "
-                 CLM_URI " TEXT, "
-                 CLM_ALBUM " TEXT, "
-                 CLM_ALBUM_ARTIST " TEXT, "
-                 CLM_YEAR " INTEGER, "
-                 CLM_RATING " INTEGER, "
-                 CLM_GENRE " TEXT, "
-                 CLM_GROUPING " TEXT, "
-                 CLM_TRACKNUMBER " INTEGER, "
-                 CLM_DATEADDED " INTEGER, "
-                 CLM_BPM " INTEGER, "
-                 CLM_BITRATE " INTEGER, "
-                 CLM_COMMENT " TEXT, "
-                 CLM_PLAYCOUNT" INTEGER, "
-                 CLM_COMPOSER " TEXT, "
-                 CLM_PREVIEW " TEXT)");
+        QString strQuery(
+                "CREATE TEMP TABLE IF NOT EXISTS %1"
```

Comment:
I had to write string-variable instead of variable-string for clang-format, in case anyone wants to ask. Suggestions for doing it better are welcome ^^

Reply:
`QStringLiteral("CREATE TEMP TABLE IF NOT EXISTS ") + m_tempTableName + QStringLiteral( [...] )`

Reply:
Like this? bbabfad11c

Reply:
I think you can put string literals and preprocessor string literals into the same qstringliteral (without + in between).

Reply:
Which will probably make clang-format reformat it to mess again, that is why I asked in the first place ^^

Reply:
Is there any functional advantage of using QStringLiteral here?

Reply:
Yes: https://woboq.com/blog/qstringliteral.html